launcher-implementation/src/main/scala/xsbt/boot/Update.scala

```diff
@@ -142,6 +142,9 @@ final class Update(config: UpdateConfiguration) {
           }
           val ddesc = addDependency(moduleID, app.groupID, resolvedName, app.getVersion, "default(compile)", u.classifiers)
           System.out.println("Getting " + app.groupID + " " + resolvedName + " " + app.getVersion + " " + reason + "...")
+          if (resolvedName == "sbt") {
+            System.out.println("Updating sbt, this may take some time.")
+          }
```

Comment:
I don't know if we should hardcode this to sbt. Maybe you could just add it after `reason + ". "` or something.